djangoRestReactShop/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+from django.urls import path, include
+from django.conf.urls.static import static
+from django.conf import settings
+
+urlpatterns = static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT) + [
+    path('admin', admin.site.urls),
+    path('api/v1/accounts/', include('accounts.urls')),
+    path('api/v1/products/', include('product.urls')),
+    path('api/v1/categories/', include('category.urls')),
+    path('api/v1/orders/', include('order.urls')),
```

Comment:
this is fine for now, but in future it would be great to have special directories/packages for API versions, so you url configs will look neater and it will be easier to make new API version still process old version endpoints

Reply:
понял, пасибо, почитаю